docker/Dockerfile.xenial_debug

```diff
@@ -26,3 +27,11 @@ RUN apt-get update && \
    && apt-get upgrade -y \
    && apt-get clean \
    && rm -rf  /var/log/*log /var/lib/apt/lists/* /var/log/apt/* /var/lib/dpkg/*-old /var/cache/debconf/*-old
+
+# upgrade libstdc++6
+# hadolint ignore=DL3008
```

Comment:
can we fix this instead of ignoring?

Reply:
DL3008 requires to specify version. Don't know how to specify version for `libstdc++6`